source/userguide/dvc.rst

```diff
@@ -0,0 +1,66 @@
+DVC for Model Tracking
+======================
+
+We use Data Version Control, or DVC, in Mindmeld in order to track trained models in our applications. DVC is an
+open-source version control system for data science and machine learning projects that enables versioning of large
+files and directories in concert with Git without storing the data itself in Git.
+
+You can find more info about DVC in the `official documentation <https://dvc.org/doc>`_.
+
+
```

Comment:
We should add a helpful table of currently supported DVC commands (init, save, checkout) and their description.